custom-checks/checkstyle/src/main/java/org/openhab/tools/analysis/checkstyle/readme/MarkdownCheck.java

```diff
@@ -51,6 +44,7 @@
  * info</a>
  *
  * @author Erdoan Hadzhiyusein - Initial contribution
+ * @author Lyubomir Papazov - Change the Markdown parser to flexmark
```

Comment:
The author-tags and signed off seem inconsistent

Reply:
Yes, actually @lpapazow did the changes on the checks and he's the rightful author. He's been working on a new binding and he hasn't had time to update this PR. Because of #244 , we decided that it would be a good idea to include this check in the new release, so I added a new test and fixed some comments from @dstivanov.

Reply:
AFAIK the also-by should then be added to the commit message, see https://docs.openhab.org/developers/contributing/contributing.html#sign-your-work